{PROJECT_NAME}/Sources/Data/NetworkAPI/NetworkAPI.swift

```diff
@@ -2,23 +2,19 @@
 //  NetworkAPI.swift
 //
 
-import Alamofire
 import Foundation
+import Moya
 import RxSwift
 
 final class NetworkAPI: NetworkAPIProtocol {
 
-    private let decoder: JSONDecoder
+    private let provider: MoyaProvider<MultiTarget>
```

Comment:
Please work with @suho to clarify this [comment](https://github.com/nimblehq/ios-templates/pull/394#discussion_r1035608474)

Reply:
@Shayokh144 Regarding [this discussion](https://nimble-co.slack.com/archives/C418DBL2W/p1670300978623669), it's better to remove `NetworkAPI` and `NetworkAPIProtocol` in this PR